src/pages/trade/InnerPanel.tsx

```diff
@@ -41,12 +38,16 @@ export const InnerPanel = ({
       onValueChange={setValue}
       items={[
         {
-          content: <TvChart />,
+          content: launchableMarketId ? (
+            <TvChartLaunchable marketId={launchableMarketId} />
```

Comment:
I did two separate implementations because the `<TvChart>` has additional hooks for existing markets. Instead of introducing logic to `disable` those hooks, I figured a completely separate implementation would be easier than having additional branching logic in each hook

Reply:
oh yeah i think separate components are fine, but my suggestion was to standardize how each of these components gets `marketId` - one of them is from a prop, and one of them is a hook into redux right now, so to me it makes sense to move both of them to a prop, but also nbd